store/models.py

```diff
@@ -0,0 +1,105 @@
+from django.db import models
+from autoslug.fields import AutoSlugField
+from django.core.validators import MaxValueValidator, MinValueValidator
+from ordered_model.models import OrderedModel
+from django.contrib.auth import get_user_model
+
+
+
+
+class Category(models.Model):
+    name = models.CharField(max_length=100)
+    name_ar = models.CharField(max_length=150)
+    slug = AutoSlugField(max_length=100, unique=True, populate_from="name")
+    img = models.ImageField(blank=True)
+
+    class Meta:
+        ordering = ["name"]
```

Comment:
@m7mdGNo why do we order by name ?
We want to order by order

Reply:
@m7mdGNo  You can add created field and order with it.